letsencrypt/data/ir_cron.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <openerp>
     <data noupdate="1">
-        <record id="cronjob" model="ir.cron">
+        <record id="letsencrypt_cron" model="ir.cron">
```

Comment:
if you insist on renaming the xmlid, we'll need a migration script for updates to go smoothly. I don't see a reason to do so to be honest, because the full xmlid will be `letsencrypt.cronjob`, which sound perfectly intelligible to me

Reply:
I rename it, to don't use a too general name. I can return it. It's no problem.